package.json

```
@@ -19,7 +19,8 @@
   },
   "bin": {
     "ota-lint-declarations": "./bin/lint-declarations.js",
-    "ota-validate-declarations": "./bin/validate-declarations.js"
+    "ota-validate-declarations": "./bin/validate-declarations.js",
+    "ota": "./bin/ota.js"
```

I foresee that we'll want to expose options rather than several executables… maybe it would be more future-proof to not take the `ota` name in those conditions, and instead keep the same convention as the other names?
```suggestion
    "ota-track": "./bin/track.js"
```